### ISSUE TITLE
Requirements for cmg connection point role?

### ISSUE THREAD

**Issue by mbiebric** [MicrosoftDocs/SCCMdocs#2267]
Hi! I deployed the cmg connection point role (only) to a new site server (MECM 1910 (5.0.8913.1000)). The connection point just stayed disconnected from a functioning cmg. The log file sms_cloud_proxyconnector.log showed: "missing role certificate. reload in next cycle" every 60s. I ended up installing the mp role as well on the same server, and the cmg cp started working as intended. The certificate store on the site server has now a "cloud proxy connector" certificate under SMS\Certificates, which wasn't there before I installed the mp role. I've removed the mp role and its prerequisites and the cmg cp is still working. We're using "enhanced http" mode for client communication.

Is it not supported to run the cmg cp role independently?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7fb371d2-b1f1-68ca-ed95-abf37e3ecf28
* Version Independent ID: cd7fc055-b430-3a4a-1002-c69087f30714
* Content: [Set up cloud management gateway - Configuration Manager](https://docs.microsoft.com/en-us/configmgr/core/clients/manage/cmg/setup-cloud-management-gateway)
* Content Source: [sccm/core/clients/manage/cmg/setup-cloud-management-gateway.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/core/clients/manage/cmg/setup-cloud-management-gateway.md)
* Product: **configuration-manager**
* Technology: **configmgr-client**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by aczechowski** [comment]
Thanks @mbiebric for taking the time to share this with the Configuration Manager documentation team. I’m sorry that you’re having problems, and I want to make sure it gets to the right people that can help. There are a couple of options available to provide the best level of support for this:
- Post on [Configuration Manager forums](https://social.technet.microsoft.com/Forums/en-US/home?category=ConfigMgrCB). These are moderated by our community MVPs who are very experienced, knowledgeable, and helpful.
- Use [Microsoft Support](https://aka.ms/cmcbsupport) to search for the issue, or open a case with professional support.
- Contact your Microsoft Premier team, such as a Premier Field Engineer or Technical Account Manager.
I'm closing this issue as there's nothing actionable for docs at this time.

One quick thought, see the article on CMG certificates, specifically this section on the [Client authentication certificate](https://docs.microsoft.com/configmgr/core/clients/manage/cmg/certificates-for-cloud-management-gateway#bkmk_clientauth). It has a note about how that cert is required on the CMG connection point.

**Comment by mbiebric** [comment]
Thanks for the quick reply. The issue is resolved in my case, but I wonder whether it's a bug. I'll try the TechNet/Tech Community forum.

Regarding the doc you referring to, it states:
"If clients use Azure AD authentication, or you configure the management point for Enhanced HTTP, this certificate isn't required."

**Comment by ARDxAdmin** [comment]
I found a way to make this occur on a stand alone server and speed the process. It appears there is a 6 hour window before the Cloud Proxy Certificate is issued and communication occurs. In SCCM Open "Configuration Manager Service Manager" and find your primary Under Components I cycled SMS_CERTIFICATE_MANAGER and SMS_CLOUD_SERVICES_MANAGER and the certificate instantly appeared in my stand alone Cloud Connection Point. You will also notice that on that server the mpfdm.log will show errors as that server is trying to reach back to the primary to verify the files from the connectors outbox hits the primaries inbox. If you create a SQL logon for the Cloud Connector machine with public and read rights to the database, specifically the primary, it clears those errors up.
